NYPLAudiobookToolkit/Core/TimeIntervalStringHelper.swift

```diff
@@ -12,8 +12,8 @@ import UIKit
 /// into a human readable string. The format will be
 /// HH:MM:SS if the TimeInterval is longer than 1 hour,
 /// otherwise it will be MM:SS.
-class HumanReadableTimestamp {
-    lazy var value: String = {
+class TimeIntervalStringHelper {
```

Comment:
non-blocking: typically naming things `Helper` ought be considered a code smell. I stand by the original name of this class because it's responsibility is to take a `TimeInterval` and transform it into a format humans can read instead of computers. If you want to rename it, I think `HumanReadableTimeInterval` makes more sense because it more accurately describes the transformation. 

more detail:
https://softwareengineering.stackexchange.com/a/247277

Reply:
I mostly fell into a trap of trading one smell for another. Not at all unlike Febreeze.
I felt a little weird about "human readable" when I considered that a "x:x" timecode is more standard to industry than something that might be more lengthy, but understandable.

Back on topic, the change that's still commencing is a need to have a representation that reads as "x hr y min remaining", and my proposal to you is that I'll revert the name of the class and then rename the two properties that will give two different representations.